src/components/landing/listing.js

```diff
@@ -5,15 +5,19 @@ import styles from "./listing.module.css";
 function Listing({ data }) {
   return (
     <>
-       <article className={styles.listing}>
+      <article className={styles.listing}>
         <div className={styles.listing__image__container}>
-          <Image src="/image-placeholder.png" width={50} height={50} alt={`${data.name} avatar`} />
+          <Image
+            src="/image-placeholder.png"
+            width={50} height={50}
```

Comment:
Its best to not hard code numbers in your code. Please move these to a variable with an appropriate name at the top of the file